README.md

```diff
@@ -128,6 +128,7 @@ from gherkin.pickles.compiler import Compiler
 
 parser = Parser()
 gherkin_document = parser.parse("Feature: ...")
+gherkin_document["uri"] = "uri_of_the_feature.feature"
```

Comment:
🔍 Looks good! Thanks!

Minor question, not a blocker: Is `"uri_of_the_feature.feature"` an acceptable "uri", here?

- If yes, that's cool,
- If not, perhaps we can use something that would be a real URI, (e.g. `"file:///home/alice/projects/my_app/uri_of_the_feature.feature"`, `"http://example.com/gherkin_features/uri_of_the_feature.feature"`)

(Minor question #2: can Parser.parse dig out "uri" out of given text to parse? Sounds improbable.)

Reply:
I copied this URL from another language's example. We can pick an example url and change it everywhere.

Reply:
No need to, change, let's merge this improvement as it stands , thanks for filling me in.